conformance/Makefile

```diff
@@ -36,16 +44,20 @@ install-nkg: ## Install NKG with provisioner on configured kind cluster
 	kubectl apply -f ../deploy/manifests/rbac.yaml
 	kubectl apply -f ../deploy/manifests/gatewayclass.yaml
 	kubectl apply -f ../deploy/manifests/service/nodeport.yaml
-	kubectl apply -f provisioner/provisioner.yaml
-
-.PHONY: update-test-kind-config 
-update-test-kind-config: ## Update kind config
-	sed -ir "s|server:.*|server: https://kind-control-plane:6443|" $(KIND_KUBE_CONFIG_FOLDER)/config
+	yq '(select(di != 3))' provisioner/provisioner.yaml | kubectl apply -f -
```

Comment:
Can we do this as part of `prepare-nkg` so we don't need to pass the prefix and tag to this command? Also, it would be nice to run the conformance tests locally against edge without building and loading the edge image.

Reply:
I've reconfigured the make command targets to create separate commands for locally built images and edge, and removed the need to call `prepare` and `install`, favouring a single command to do both.